app/src/main/java/net/kuama/documentscanner/extensions/Context.kt

```diff
@@ -8,9 +8,9 @@ import java.io.File
 internal val Context.outputDirectory: File
     get() {
         val appContext = applicationContext
-        val mediaDir = externalMediaDirs.firstOrNull()?.let {
+        val mediaDir = filesDir.let {
```

Comment:
`filesDir?.let` need a ?
otherwise it is the same of `File(filesDir, appContext.resources.getString(R.string.app_name)).apply { mkdirs() }`